### PR TITLE
[Kaptain] requirements section

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-89293

## Description of changes being made

The requirement section was ambiguous (did not stress enough that those were bare minimum requirements). We also decided we needed better reference values and examples. Following a topic-based approach to technical writting, I decided to make this its own section/page.

### Preview

http://docs-d2iq-com-pr-4471.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+beta: false
+enterprise: false
+---
+
+The amount of resources required for your project **depends on the workload** you want to run, you need to consider: 
+
+-   The number, complexity, and size of your training jobs, notebook servers, deployed models, etc.
+-   The amount of metadata and log data stored with each run.
+
+To estimate the amount of resources required for your project, it is recommended that you test your specific use case.
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:16]: @bruced2iq, @dmennell is there anything else here we could mention to help customers? How do customers currently estimate their resources? Maybe we could include 1 or 2 lines on that here?

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 24 cores                             |
+| RAM per node           | 16 GiB               | 86 GiB                               |
+| Storage per node       | 60 GiB               | 61 GiB                               |
+
```
Comment (alembiewski) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:30]: based on @andresmascl's [measurements](https://docs.google.com/spreadsheets/d/1VoF4MCsiUD2NnogcPZ-2wSsco1auZY-j91o_nHZPxMY/edit#gid=1219806816), I can see CPU requests are doubled and memory requests are x4.  Storage increase by 1 GB is rather confusing and can be ignored. Adjusting the numbers to the closest "power of two" values:
```suggestion

| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
|------------------------|----------------------|--------------------------------------|
| Number of worker nodes | at least 3           | at least 3                           |
| CPU per node           | 8 cores              | 16 cores                             |
| RAM per node           | 16 GiB               | 64 GiB                               |
| Storage per node       | 60 GiB               | 60 GiB                               |

```

File: pages/dkp/kaptain/2.1.0/install/requirements/requirements [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. Running any real world machine learning workload on Kaptain bumps these requirements. For this reason, the number of CPUs, GPUs, RAM, and persistent disk space **must be increased considerably**. 
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 24 cores                             |
+| RAM per node           | 16 GiB               | 86 GiB                               |
+| Storage per node       | 60 GiB               | 61 GiB                               |
```
Comment (alembiewski) [inline pages/dkp/kaptain/2.1.0/install/requirements/requirements:29]:
```suggestion
| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
|------------------------|----------------------|--------------------------------------|
| Number of worker nodes | at least 3           | at least 3                           |
| CPU per node           | 8 cores              | 16 cores                             |
| RAM per node           | 16 GiB               | 64 GiB                               |
| Storage per node       | 60 GiB               | 60 GiB                               |
```

**Review by alembiewski — COMMENTED** [review]
Looks good overall, left a few suggestions

**Comment by andresmascl** [comment]
> Following a topic-based approach to technical writing, I decided to make this its own section/page.
I think this was much needed because, given the nature of the system, resource requirements are in fact a topic on its own.
> 
> ### Preview
> http://docs-d2iq-com-pr-4471.s3-website-us-west-2.amazonaws.com/
Don't know about you, but I can't see the preview.

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+The amount of resources required for your project **depends on the workload** you want to run, you need to consider: 
+
+-   The number, complexity, and size of your training jobs, notebook servers, deployed models, etc.
+-   The amount of metadata and log data stored with each run.
+
+To estimate the amount of resources required for your project, it is recommended that you test your specific use case.
+
+To help you in this task, the following table provides an account of the resources required for two common operations: [installing Kaptain][install], and running a complete machine learning workflow using Tensorflow. This workflow corresponds to the [Kaptain SDK with Tensorflow tutorial][tensorflow], and includes the training, tuning, and deploying stages of a machine learning model used to recognize handwritten digits.
+
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. Running any real world machine learning workload on Kaptain bumps these requirements. For this reason, the number of CPUs, GPUs, RAM, and persistent disk space **must be increased considerably**.
```
Comment (cneth) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md]:
```suggestion
The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP.   These are minimums; 'real-world' machine learning workloads will require substantially more of all the resource types.
```

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 16 cores                             |
+| RAM per node           | 16 GiB               | 64 GiB                               |
+| Storage per node       | 60 GiB               | 60 GiB                               |
+
+<p class="message--note"><strong>NOTE: </strong>Only NVIDIA GPU instances are supported.</p>
+
+The [Kaptain installation][install] example illustrates the resources required to install Kaptain and its dependencies without running any type of workload. In other words, the **bare minimum** resources. These resources must be **freely available** within the cluster for the installation to succeed.
```
Comment (cneth) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:33]:
```suggestion
The [Kaptain installation][install] column lists the resources required to install Kaptain and its dependencies without running any workload. In other words, the **bare minimum** amount of resources. These resources must be **freely available** within the cluster for the installation to succeed.
```

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+The [Kaptain SDK with Tensorflow tutorial][tensorflow] example illustrates the resources required to install Kaptain and run the Kaptain SDK with Tensorflow tutorial. Again, these resources must be **freely available** within the cluster, and should not be allocated to any other applications.
```
Comment (cneth) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:35]:
```suggestion
The [Kaptain SDK with Tensorflow tutorial][tensorflow] column the resources required to install Kaptain and run the Kaptain SDK with Tensorflow tutorial. Again, these resources must be **freely available** within the cluster, and should not be allocated to any other applications.
```

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+For **on premise** installations, horizontal scalability is limited by the overall size of the cluster and its quotas.
```
Comment (cneth) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:37]:
```suggestion
For **on premises** installations, horizontal scalability is limited by the overall size of the cluster and its quotas.
```

**Review by cneth — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. These are minimums; 'real-world' machine learning workloads will require **substantially more** of all the resource types.
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 16 cores                             |
+| RAM per node           | 16 GiB               | 64 GiB                               |
```
Comment (andresmascl) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:28]:
```suggestion
| RAM per node           | 16 GiB               | 32 GiB                               |
```

**Review by andresmascl — CHANGES_REQUESTED** [review]
Just updating this value as per [more accurate measurements](https://docs.google.com/spreadsheets/u/1/d/1VoF4MCsiUD2NnogcPZ-2wSsco1auZY-j91o_nHZPxMY/edit#gid=308671285)

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md (hunk 8 above)
Comment (gvasquezvargas) [reply pages/dkp/kaptain/2.0.0/install/requirements/index.md:28]: And this applies to 2.0.0 and 2.1.0, correct?

Comment (alembiewski) [reply pages/dkp/kaptain/2.0.0/install/requirements/index.md:28]: Yes

File: pages/dkp/kaptain/2.1.0/install/requirements/index.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. These are minimums; 'real-world' machine learning workloads will require **substantially more** of all the resource types.
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 16 cores                             |
+| RAM per node           | 16 GiB               | 64 GiB                               |
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.1.0/install/requirements/index.md:28]:
```suggestion
| RAM per node           | 16 GiB               | 32 GiB                               |
```

**Review by andresmascl — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. These are minimums; 'real-world' machine learning workloads will require **substantially more** of all the resource types.
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 16 cores                             |
+| RAM per node           | 16 GiB               | 32 GiB                               |
+| Storage per node       | 60 GiB               | 60 GiB                               |
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:29]: @alembiewski @andresmascl as suggested by Dan, is it ok if we bump this to:
```suggestion
| Storage per node       | 120 GiB               | 120 GiB                               |
```

File: pages/dkp/kaptain/2.1.0/install/requirements/index.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. These are minimums; 'real-world' machine learning workloads will require **substantially more** of all the resource types.
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 16 cores                             |
+| RAM per node           | 16 GiB               | 32 GiB                               |
+| Storage per node       | 60 GiB               | 60 GiB                               |
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.1.0/install/requirements/index.md:29]: @alembiewski @andresmascl as suggested by Dan, is it ok if we bump this to:
```suggestion
| Storage per node       | 120 GiB               | 120 GiB                               |
```

File: pages/dkp/kaptain/2.0.0/install/requirements/index.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+<p class="message--note"><strong>NOTE: </strong>Only NVIDIA GPU instances are supported.</p>
+
+The [Kaptain installation][install] column lists the resources required to install Kaptain and its dependencies without running any workload. In other words, the **bare minimum** amount of resources. These resources must be **freely available** within the cluster for the installation to succeed.
+
+The [Kaptain SDK with Tensorflow tutorial][tensorflow] column lists the resources required to install Kaptain and run the Kaptain SDK with Tensorflow tutorial. Again, these resources must be **freely available** within the cluster, and should not be allocated to any other applications.
+
+For **on premises** installations, horizontal scalability is limited by the overall size of the cluster and its quotas.
```
Comment (msdolbey) [inline pages/dkp/kaptain/2.0.0/install/requirements/index.md:37]:
```suggestion
For **on-premises** installations, horizontal scalability is limited by the overall size of the cluster and its quotas.
```

File: pages/dkp/kaptain/2.1.0/install/requirements/index.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+<p class="message--note"><strong>NOTE: </strong>Only NVIDIA GPU instances are supported.</p>
+
+The [Kaptain installation][install] column lists the resources required to install Kaptain and its dependencies without running any type of workload. In other words, the **bare minimum** resources. These resources must be **freely available** within the cluster for the installation to succeed.
+
+The [Kaptain SDK with Tensorflow tutorial][tensorflow] column lists the resources required to install Kaptain and run the Kaptain SDK with Tensorflow tutorial. Again, these resources must be **freely available** within the cluster, and should not be allocated to any other applications.
+
+For **on premises** installations, horizontal scalability is limited by the overall size of the cluster and its quotas.
```
Comment (msdolbey) [inline pages/dkp/kaptain/2.1.0/install/requirements/index.md:37]:
```suggestion
For **on-premises** installations, horizontal scalability is limited by the overall size of the cluster and its quotas.
```

**Review by msdolbey — APPROVED** [review]
👍 
Some really small nits - but looks good to me.

It is required

**Pull request merged**